.github/workflows/cla.yml

```diff
@@ -0,0 +1,43 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+# explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
+permissions:
+  actions: write
+  contents: read
+  pull-requests: write
+  statuses: write
+
+jobs:
+  CLAAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        uses: contributor-assistant/github-action@v2.4.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.CLA_TOKEN }}
+        with:
+          path-to-signatures: 'signatures/version1/signatures.json'
+          path-to-document: 'https://gist.github.com/af3262c2eb70b14b8e96260e8acfc2cf' # CLA document
```

Comment:
This is a raw text file that's hard to read—I propose we add a Markdown-formatted version in MaterializeInc/cla and then link to that nicely rendered version here.

Reply:
https://github.com/MaterializeInc/cla/blob/main/Materialize_CLA.md looks good as a markdown file with same content as the gist.